src/utils/query.js

```diff
@@ -31,7 +31,7 @@ const handleResponse = (endpoint, res, placeholder) => {
 	return res.data;
 };
 
-export default async (client, endpoint, parameters) =>
+const query = async (client, endpoint, parameters) =>
```

Comment:
change this to `export const query` and change all the reference?

Reply:
@shuse2 I was doing that earlier but I saw that I will have to change the reference in a lot of files in `/src `and `/test` so instead of that I'm just exporting query as default `export default query` and handler as named export so that no changes are required in other files where they are importing `query` as default.

Reply:
I think it's better to not to have `default` in general, but the eslint complains.
It would be helpful when we convert this to typescript too =)